### PR TITLE
Database queries

### PR DESCRIPTION
**Pull request by Konstantsiy**

### CODE REVIEW

File: scripts/queries.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+group by r.user_id order by c;
+
+
+-- get the list of users which did not request to compress images
+select id from converter.users
+where id not in (
+    select u.id from converter.users u
+    join converter.requests r
+    on u.id = r.user_id and r.target_format = 'jpeg'
+);
```
Comment (papahmsolo) [inline scripts/queries.sql:66]: What if the conversion is from jpeg to png? Will this still work?

**Review by papahmsolo — CHANGES_REQUESTED** [review]
👀

File: scripts/queries.sql (hunk 1 above)
Comment (Konstantsiy) [reply scripts/queries.sql:66]: > What if the conversion is from jpeg to png? Will this still work?

But the request description says that we need users who did not make a request for data compression, that is, it is only PNG -> JPEG, if I understand correctly. In addition, in the "image/png" library itself, there is no parameter for compression, as in the library for jpg.

Comment (Konstantsiy) [reply scripts/queries.sql]: I misunderstood, sorry

**Review by papahmsolo — APPROVED** [review]
👍

**Pull request merged**